packages/app/src/App.tsx

```diff
@@ -225,11 +226,13 @@ const routes = (
 
 const App = () => (
   <AppProvider>
-    <AlertDisplay />
-    <OAuthRequestDialog />
-    <AppRouter>
-      <Root>{routes}</Root>
-    </AppRouter>
+    <LayoutProvider>
```

Comment:
I'd say we need to find another way to do this, it's not quite worth this kind of breaking change. We ofc have the app providers, only tricky bit is that that's from `core-app-api` and we're making additions to `core-components`, so need to figure that part out

Reply:
I'm not sure if it's a breaking change, as we have it now the `<LayoutProvider>` could be optional, I make sure that if you don't add this provider the `focusComponent` gonna be noop. and the sidebar "skip" will not be shown 

I've tried adding the LayoutProvider inside the AppProvider, but that creates a dependency between core-app-api and core-components and that is something that we want to avoid right?